connector/src/main/java/org/jboss/as/connector/subsystems/common/pool/Constants.java

```diff
@@ -145,7 +145,6 @@
             .build();
 
     public static PropertiesAttributeDefinition CAPACITY_INCREMENTER_PROPERTIES = new PropertiesAttributeDefinition.Builder(CAPACITY_INCREMENTER_PROPERTIES_NAME, true)
-            .setXmlName(org.jboss.jca.common.api.metadata.common.Extension.Tag.CONFIG_PROPERTY.getLocalName())
```

Comment:
I would feel more comfortable if there was some coverage of this in the datasources subsystem tests